syntax = "proto3";
package base_package;

 //定义服务,用在rpc传输中
service FormatData {
  rpc DoFormat(actionrequest) returns (actionresponse){}
}
message actionrequest {
  string text = 1;
}
message actionresponse{
  string text=1;
}